src/HasteModuleLoader/HasteModuleLoader.js

```diff
@@ -591,7 +591,7 @@ Loader.prototype._shouldMock = function(currPath, moduleName) {
       this._configShouldMockModuleNames[moduleName] = true;
       for (var i = 0; i < this._unmockListRegExps.length; i++) {
         unmockRegExp = this._unmockListRegExps[i];
-        if (unmockRegExp.test(modulePath)) {
+        if (unmockRegExp.test(modulePath) || modulePath === null) {
```

Comment:
Won't this just never mock node built-ins?

Instead, shouldn't this be something like
`unmockRegExp.test(modulePath) || (modulePath === null && unmockRegExp.test(moduleName)`?

Otherwise I think this will just always mark node built-ins as unmocked, right?